src/puptoo/upload.py

```diff
@@ -0,0 +1,35 @@
+import io
+import json
+from minio import Minio
+from .utils import config, puptoo_logging
+
+logger = puptoo_logging.initialize_logging()
+
+
+# Minio client setup and upload yum_updates object
+def upload_object(yum_updates, extra, msg):
+    _bytes = json.dumps(yum_updates).encode("utf-8")
+    client = Minio(endpoint=config.S3_ENDPOINT, access_key=config.ACCESS_KEY, secret_key=config.SECRET_KEY, secure=config.USE_SSL)
+    if client.bucket_exists(config.BUCKET_NAME):
+        try:
+            client.put_object(bucket_name=config.BUCKET_NAME, object_name=extra["request_id"], data=io.BytesIO(_bytes), length=len(_bytes))
```

Comment:
if the request_id is `80b49bdd608c/JcLwFQ4WI6-000007` then it creates the directory with name `80b49bdd608c` and inside it json file with name `JcLwFQ4WI6-000007` is created. Ideally it should create the file in bucket with name `80b49bdd608c/JcLwFQ4WI6-000007` and not the directory.

Reply:
Thanks @SteveHNH for confirming!